client/dashboard/sites/index.tsx

```diff
@@ -146,3 +150,63 @@ export default function Sites() {
 		</>
 	);
 }
+
+function recordViewChanges(
```

Comment:
Can we maybe move this function to `views.ts`? It's meant to co-locate all view-related logic 🤔

Reply:
Good point. This is yet another function that iterates over the uber View object.